change/@internal-calling-stateful-client-e108bf17-9cc5-4189-9c8f-66acc0b48938.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This PR removes some API that should have only been in the beta build.

So really "undoes" a minor change.